DejaGnu has a hack for locating the shared directory that doesn't
work on Gobo.  So extend the hack to work on Gobo too... fixing
it would be better though, because it might could be accepted upstream.

diff -Naur dejagnu-1.4.4/runtest dejagnu-1.4.4.new/runtest
--- dejagnu-1.4.4/runtest	2003-08-16 21:27:59.000000000 -0400
+++ dejagnu-1.4.4.new/runtest	2007-12-30 09:24:50.000000000 -0500
@@ -96,7 +96,8 @@
 #
 # which is a very weak assumption
 #
-for i in `echo ${execpath} | sed  -e 's@/[^/]*$@/share/dejagnu@'` `echo ${execpath} | sed  -e 's@/[^/]*/[^/]*$@/share/dejagnu@'` $execpath ; do
+#prefix, or goboShared? wait, shared is always shared
+for i in @%Compile_goboShared%@/dejagnu `echo ${execpath} | sed  -e 's@/[^/]*$@/share/dejagnu@'` `echo ${execpath} | sed  -e 's@/[^/]*/[^/]*$@/share/dejagnu@'` $execpath ; do
     if expr $verbose \> 1 > /dev/null ; then
 	echo Looking for $i/runtest.exp.
     fi
